index.html

```diff
@@ -15,14 +15,22 @@
             <nav class="logo-show">
               <h2>My Logo</h2>
             </nav>
-            <nav class="menu">
-              <nav><img src="photo/navs/Union.png" alt=""></nav>
-            </nav>
+            <div class="menu">
+             <button class="btn-menu"><img src="photo/navs/Union.png" alt=""></button> 
+            </div>
+            <div class="t-menu">
+              <button class="c-button">&times;</button>
+              <ul class="tul">
+                <li class="tul-1">Portfolio</li>
+                <li class="tul-1">About</li>
+                <li class="tul-1">Contact</li>
```

Comment:
- [ ] Please I think that the portfolio link should take the user to the recent works sections, so kindly fix this. 😁 you can wrap it with `a` tag and include href to the sections with the id 💪🏻💪🏻💪🏻✅✅